diff --git a/chrome/installer/linux/debian/update_dist_package_versions.py b/chrome/installer/linux/debian/update_dist_package_versions.py
index a3d824eeabbc4b54f52392995ebadaaea1f2171c..52376db86c8ebd5cc4bb00d7b89f732b757b08f1 100755
--- a/chrome/installer/linux/debian/update_dist_package_versions.py
+++ b/chrome/installer/linux/debian/update_dist_package_versions.py
@@ -24,7 +24,6 @@ SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
 SUPPORTED_DEBIAN_RELEASES = {
     'Debian 8 (Jessie)': 'jessie',
     'Debian 9 (Stretch)': 'stretch',
-    'Debian 10 (Buster)': 'buster',
 }
 
 SUPPORTED_UBUNTU_RELEASES = {
