--- update_package_provides.py	2018-08-14 15:39:47.098846999 +0000
+++ update_package_provides.py	2018-08-14 15:38:35.770846999 +0000
@@ -86,6 +86,14 @@
         # overwrite the originals.
         "https://download.opensuse.org/update/leap/%s/oss/" % version,
   ]
+rpm_sources['Fedora ' + '26'] = [
+    "https://download.fedoraproject.org/pub/fedora/linux/releases/26/Everything/x86_64/os/",
+    "https://pubmirror1.math.uh.edu/fedora-buffet/archive/fedora/linux/updates/26/x86_64/",
+]
+rpm_sources['Fedora ' + '25'] = [
+    "https://download.fedoraproject.org/pub/fedora/linux/releases/25/Everything/x86_64/os/",
+    "https://mirrors.rit.edu/fedora/archive/fedora/linux/updates/25/x86_64/",
+]
 
 provides = {}
 for distro in rpm_sources:
