diff --git a/chrome/installer/linux/rpm/update_package_provides.py b/chrome/installer/linux/rpm/update_package_provides.py
index 7842464740a9ead296207218db64d5be9f0401fc..edb3026f3b10d1113f4b12b119dddf038e313ebb 100755
--- a/chrome/installer/linux/rpm/update_package_provides.py
+++ b/chrome/installer/linux/rpm/update_package_provides.py
@@ -81,6 +81,14 @@ for version in SUPPORTED_OPENSUSE_LEAP_RELEASES:
         # overwrite the originals.
         "https://download.opensuse.org/update/leap/%s/oss/" % version,
   ]
+rpm_sources['Fedora ' + '26'] = [
+    "https://download.fedoraproject.org/pub/fedora/linux/releases/26/Everything/x86_64/os/",
+    "https://pubmirror1.math.uh.edu/fedora-buffet/archive/fedora/linux/updates/26/x86_64/",
+]
+rpm_sources['Fedora ' + '25'] = [
+    "https://download.fedoraproject.org/pub/fedora/linux/releases/25/Everything/x86_64/os/",
+    "https://mirrors.rit.edu/fedora/archive/fedora/linux/updates/25/x86_64/",
+]
 
 provides = {}
 for distro in rpm_sources:
